src/components/MovieDetails/index.tsx

```diff
@@ -110,6 +111,13 @@ const MovieDetails = ({ movie }: MovieDetailsProps) => {
     mutate: revalidate,
   } = useSWR<MovieDetailsType>(`/api/v1/movie/${router.query.movieId}`, {
     fallbackData: movie,
+    refreshInterval: refreshIntervalHelper(
+      {
+        downloadStatus: movie?.mediaInfo?.downloadStatus,
+        downloadStatus4k: movie?.mediaInfo?.downloadStatus4k,
+      },
+      5000
```

Comment:
5 seconds interval is way too fast considering that a user can only modify the job schedule to go as fast as 30 seconds in the UI. 
30 or even 15 seconds would be good, though I'm leaning towards the former since IMO a user would need to be actively looking at the progress bar and comparing it to their *arr instance to notice the difference between 30 seconds and 15 seconds interval.

Reply:
I'm okay with changing it to 15 seconds. I feel like any higher might be too long especially since the default is 60.

Reply:
Sounds good 👍